reana_job_controller/config.py

```diff
@@ -80,3 +84,15 @@
 """Kerberos configMap name. Must be the same as in
 reana_cluster/backends/kubernetes/templates/configmaps/kerberos.yaml.
 """
+
+SLURM_HEADNODE_HOSTNAME = os.getenv('SLURM_HOSTNAME', 'hpc-batch.cern.ch')
```

Comment:
Success for RooFit demo run on CERN Slurm! :tada: 

Perhaps the hpc-batch and partition etc should be defined not as part of `config.py`, but inside the CERN-specific class?  Might be easier to create other classes for other Slurm clusters. (We can also tackle this later due to zeitnot.)